docs/release_notes.rst

```diff
@@ -34,6 +34,12 @@ developers with their pysat-based packages for data handling or analysis.
 
 v2.3 Release
 ------------
-Might as well draft something up here
+The Python Satellite Data Analysis Toolkit (pysat) v2.3.0 release is intended
+to be the last release of the v2.x series and the last release to
+include support for Python 2.x. pysat v2.3.0 is primarily intended
+to provide DeprecationWarnings for pysat functions that are changing as part
+of the 3.0.0 release. In addition, version restrictions have been added
+to all packages to help ensure this release remains installable for future
+users looking for legacy support.
```

Comment:
Are we going to cap these now or as a patch if it comes up?  See discussion in #747

Reply:
Ah, good catch, I was behind and thought that was humming along. I'll update.